tool/resources/org/antlr/v4/tool/templates/codegen/Go/Go.stg

```diff
@@ -702,8 +700,7 @@ ListLabelName(label) ::="<label>"
 CaptureNextToken(d) ::= "<d.varName> = p.GetTokenStream().LT(1)"
 CaptureNextTokenType(d) ::= "<d.varName> = p.GetTokenStream().LA(1);"
 
-StructDecl(struct, ctorAttrs, attrs, getters, dispatchMethods, interfaces, extensionMembers, tokenDecls, tokenTypeDecls,
-	tokenListDecls, ruleContextDecls, ruleContextListDecls, attributeDecls, superClass={ParserRuleContext}) ::= <<
+StructDecl(struct, ctorAttrs, attrs, getters, dispatchMethods, interfaces, extensionMembers, tokenDecls, tokenTypeDecls, tokenListDecls, ruleContextDecls, ruleContextListDecls, attributeDecls, superClass={ParserRuleContext}) ::= <<
```

Comment:
I totally get it. :) I tend to avoid the problem by breaking up things that have that many params, but I couldn't do that here. If this were complex code and a lot of the params were used, I would have done as you suggest, but since it's a relatively simple template, and only one param is used, it didn't seem like much of a burden. If you feel strongly about it, I'm more than happy to change it back!